.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,456 @@
+          echo "::set-env name=TEST_PHP_EXECUTABLE::${env:PHPROOT}\php.exe"
+          echo "::set-env name=BUILD_TYPE::${{ matrix.build_type }}"
+          echo "::set-env name=VC_VERSION::${{ matrix.vc_num }}"
+          echo "::set-env name=PHP_ARCH::${{ matrix.arch }}"
+          echo "::set-env name=PACKAGE_PREFIX::phalcon"
+          echo "::set-env name=EXTENSION_NAME::phalcon"
+          echo "::set-env name=EXTENSION_FILE::php_phalcon.dll"
+
+      - name: Setup Composer Token
+        run: |
+          # To increase the Composer rate limit we're use GitHub authentication
+          if ("${{ secrets.COMPOSER_TOKEN }}" -ne "") {
+              composer config github-oauth.github.com ${{ secrets.COMPOSER_TOKEN }}
+          }
+
+      - name: Setup Common Environment
+        run: |
+          Import-Module .\.ci\win-ci-tools.psm1
+          SetupCommonEnvironment
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Setup Cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Download phalcon pecl package
+        uses: actions/download-artifact@v1
+        with:
+          name: phalcon-pecl
+
+      - name: Install PHP SDK Binary Tools
+        run: |
+          Import-Module .\.ci\win-ci-tools.psm1
+          InstallPhpSdk
+
+      - name: Install PHP Dev pack
+        run: |
+          Import-Module .\.ci\win-ci-tools.psm1
+          InstallPhpDevPack
+
+      - name: Download PHP src
+        run: |
+          Import-Module .\.ci\win-ci-tools.psm1
+          DownloadPhpSrc
+
+      - name: Getting Details About Installed PHP
+        run: |
+          Import-Module .\.ci\win-ci-tools.psm1
+          Get-Php "${env:PHPROOT}"
+
+      - name: Install System Dependencies
+        run: choco install -y --cache-location=C:\Downloads\Choco re2c
+
+      - name: "Setup Visual Studio Command Line for PHP SDK ${{ matrix.arch }}"
+        run: .ci\vsenv.bat -arch=${{ matrix.arch }} -host_arch=${{ matrix.arch }}
+
+      - name: Fix Environment Variables
+        shell: powershell
+        run: |
+          $v = "${env:WindowsSDKVersion}" -replace '\\$', ''
+          Write-Output "::set-env name=WindowsSDKVersion::$v"
+
+          $v = "${env:WindowsSDKLibVersion}" -replace '\\$', ''
+          Write-Output "::set-env name=WindowsSDKLibVersion::$v"
+
+      - name: Unpack package
+        run: |
+          Import-Module .\.ci\win-ci-tools.psm1
+          Expand-Item7zip "phalcon-pecl\phalcon-pecl.tgz" "${env:PHP_SRC_PATH}\ext\phalcon"
+
+      - name: Compile Phalcon
+        shell: powershell
+        run: |
+          $env:Path += ";${env:PHP_SRC_PATH}"
+          cd ${env:PHPROOT}\pecl
```

Comment:
Is this path correct?

Reply:
No. Tried a few things but didn't work out. Will download a virtual box today. Debugging in Github actions is a bit too slow.